src/cpr_data_access/parser_models.py

```diff
@@ -19,6 +19,14 @@
 logger = logging.getLogger(__name__)
 
 
+class VerticalFlipError(Exception):
+    """Exception for when a vertical flip fails."""
```

Comment:
Slightly unclear on whether to define this as an exception or an Error as we inherit from the Exception class @joel-wright